Launcher/Classes/Extensions.swift

```diff
@@ -54,3 +54,12 @@ extension Bundle {
         return path(forResource: name, ofType: ext.rawValue)
     }
 }
+
+extension Dictionary where Key == String {
+    mutating func append(dictionary: Dictionary) {
+        let resultingDictionary = dictionary
```

Comment:
Why do you shadow the `dictionary` here? I don't think that's needed.

Reply:
Because this: `self[$0.0] = $0.1` will change it, and `dictionary` is a let constant

Reply:
Will change what? `resultingDictionary` is also a `let`. It will change `self`, which it can as the function is marked as `mutating`. You shouldn't need the shadow variable.

Reply:
True.. I tried to do smth else before and it confused me :)